week01/3Sum.py

```diff
@@ -0,0 +1,22 @@
+ class Solution:
+    # TC: O(n^2)
+    # MC: O(n)
+    def threeSum(self, nums: List[int]) -> List[List[int]]:
+        triplets = set()
```

Comment:
This leads the solution to O(N^3) in worst case. set is not ALWAYS O(1). It's the average cost.